addons/config/webhooks/MutatingWehookConfiguration.yaml

```diff
@@ -0,0 +1,49 @@
+
+---
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: MutatingWebhookConfiguration
+metadata:
+  creationTimestamp: null
+  name: mutating-webhook-configuration
+  annotations:
+    cert-manager.io/inject-ca-from: default/my-webhook-certificate #This  tells cert manager to insert the CA bundle into each clientConfig below from default/my-webhook-certificates. A secret created on the cert-mangaer side
+webhooks:
+  # One client section per package
+  - clientConfig:
```

Comment:
The clientConfig sections here are just examples and need cleanning up. 
Basically we want to merge this file and folks can add the clientconfig sections for each one of their packages as they work on their *.go files